courses/machine_learning/deepdive2/feature_engineering/solutions/3_Basic_Feature_Engineering_in_Keras.ipynb

```diff
@@ -0,0 +1,2565 @@
+{
```

Comment:
<p>instead of this cell and the one below, I'd just do </p><p>housing_df.describe()</p><p>and then interpret the 'count' value to see nothing is missing. You also get other descriptive statistics with .describe()</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/777/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Reply:
<p>Agreed.  Removed the two cells "info()" and isnull()" and replaced with ".describe().  Also added the following comment in the cell:</p><p> </p><p># Note the count row and corresponding columns.&nbsp;&nbsp;</p><p># The count shows 20433.000000 for all feature columns.&nbsp;Thus, there are no misisng values.</p><p> </p>